apps/paf/fields/index.js

```diff
@@ -578,7 +578,7 @@ module.exports = {
   },
   'crime-location-phone': {
     className: ['govuk-input', 'govuk-input--width-20'],
-    validate: [{ type: 'maxlength', arguments: 20 }]
+    validate: ['required', 'internationalPhoneNumber', { type: 'maxlength', arguments: 25 }]
```

Comment:
Other forms (ACQ/SAP, Firearms, ASC) have 200 as the number. Do you think that's too high?

Reply:
How about 50 chars? That would seem to me to be more than enough for most use cases, and cvan always be upped if we get reports of odd cases failing due to not enough chars.

Reply:
Really for phone number ?  but normally phone length is not mor then 30 both for international and local

Reply:
Normally, yes...but it is probably best to cater to potential edge cases. 30 is probably fine too, but I don't see the issue with being at least a little more generous.